critiquebrainz/db/users.py

```diff
@@ -0,0 +1,17 @@
+from critiquebrainz import db
+from critiquebrainz.db import exceptions as db_exceptions
+import sqlalchemy
+
+
+def getusers_id(usernames):
```

Comment:
Name and documentation of this function are misleading. You don't return just an ID, but all the columns.

I would suggest to alter this function slightly. We can have more general `get_many_by_mb_username`, which would return a list of users (as dictionaries).